### DISCUSSION TITLE
Still maintained? Want help to maintain parse?

### DISCUSSION THREAD

**Discussion by nabobalis** [r1chardj0n3s/parse#193] [General]
Dear @r1chardj0n3s,

We use parse for internal use within [sunpy](https://github.com/sunpy/sunpy) and it is very important for us. While we vendor it in, we do not want to create our own fork/version of the library.

I was wondering if you would appreciate other people joining you in maintaining (reviewing pull requests, doing releases, updating the skeleton etc) this library?

I am sure there are others would be happy to help!

**Comment by wimglenn** [comment]
Hello @nabobalis, I've picked up maintenance on this project and am able make releases. Help reviewing pull requests etc is of course appreciated!
